committee-steering/governance/faq.md

```diff
@@ -0,0 +1,100 @@
+# FAQ
+
+## Why is it so long
+
+These documents attempt to ensure the institutional knowledge
+of individuals managing SIGs today is well documented and available
+to all members in the SIG.  There are a surprising number of responsibilities and
+processes that have developed as part of facilitating management
+of the project.
+
+## Why so much structure
+
+These documents intentionally leans toward more structure rather than
+less.  There are 2 reasons for this:
+
+- It is easy for SIGs to remove or edit sections in this
+  template to reduce structure, whereas it is harder to make up new
+  structure from scratch
+- The SIG governance questionnaire showed a consistent preference for
+  more structure over less.  (almost all answers were to have
+  the same or more amount of structure)
+
+## Why use mailing lists for tracking roles
```

Comment:
Per other discussions... this is not tenable, and OWNERS has to be a system of record.

Reply:
Will update